webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
```

Comment:
based on the cli argument we can handle development and production modes.
for example check our [remora plugin ](https://github.com/ProtoPie/remora-plugin/blob/main/webpack.config.js)
`mode: argv.mode === 'production' ? 'production' : 'development',`
and then in the package json something like
```
"build:dev": "webpack --configuration development",
"build": "webpack --configuration production",
```